mnemosyne/core/embeddings.py

```
@@ -95,39 +118,37 @@ def _get_prefix(kind: str) -> str:
+    cfg = get_config()
     return bool(
-        os.environ.get("MNEMOSYNE_NO_EMBEDDINGS")
-        or os.environ.get("MNEMOSYNE_SKIP_EMBEDDINGS")
-        or os.environ.get("MNEMOSYNE_EMBEDDINGS_OFF")
+        cfg.get_bool("no_embeddings")
+        or cfg.get_bool("skip_embeddings")
+        or cfg.get_bool("embeddings_off")
     )
 
 
 def _is_api_model(model_name: str) -> bool:
```

Ruff (0.15.21)

[warning] Too many return statements (7 > 6)

(PLR0911)